src/payment/strategies/paypal-commerce/paypal-commerce-payment-strategy.ts

```diff
@@ -26,7 +26,7 @@ export default class PaypalCommercePaymentStrategy implements PaymentStrategy {
         const { initializationData } = getPaymentMethodOrThrow(methodId, gatewayId);
         const { orderId, buttonStyle } = initializationData;
```

Comment:
Will we have the same issue with `buttonStyle`?

Reply:
No, if button style will be undefined, this style wont apply. That's it